source/Crew.h

```diff
@@ -0,0 +1,72 @@
+/* Crew.h
+Copyright (c) 2019 by Luke Arndt
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef CREW_H_
+#define CREW_H_
+
+#include "Ship.h"
+
+class Crew
+{
+public:
+	// Calculate one day's salaries for the Player's fleet
+	static int64_t CalculateSalaries(
+		const std::vector<std::shared_ptr<Ship>> &ships,
+		const Ship * &flagship,
```

Comment:
This should not be a reference.

Also, you need to match the style of other headers and have a single line declaration:
```c++
static int64_t CalculateSalaries(const std::vector<std::shared_ptr<Ship>> &ships, const Ship *flagship, bool includeExtras = true);
```
The style guide's suggestion of 80 columns is rarely adhered to, especially when it comes to header declarations.